package.json

```diff
@@ -494,13 +512,15 @@
     "webpack-dev": "webpack --mode development --watch"
   },
   "dependencies": {
+    "@types/opn": "^5.1.0",
     "compressing": "1.4.0",
     "deepmerge": "3.1.0",
     "fs-extra": "7.0.1",
     "globule": "1.2.1",
     "jsforce": "1.8.5",
     "klaw": "3.0.0",
     "mime-types": "2.1.21",
+    "opn": "^5.4.0",
```

Comment:
I like to go with certain versions instead of using ^ as this has broken Forcecode before